File: scratch/count_tags.py
import sys

path = r"c:\Users\seanh\Desktop\Sean\Claude Cowork Projects (MP)\Art Learning Tutor\socratic-tutor\src\app\instructor\[sessionId]\analysis\page.tsx"

with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

print(f"File length: {len(content)}")
print(f"Braces {{: {content.count('{')}")
print(f"Braces }}: {content.count('}')}")
print(f"Parens (: {content.count('(')}")
print(f"Parens ): {content.count(')')}")
print(f"Frag <>: {content.count('<>')}")
print(f"Frag </>: {content.count('</>')}")
print(f"Div <div>: {content.count('<div')}")
print(f"Div </div>: {content.count('</div>')}")

File: scratch/count_tags.js
const fs = require('fs');
const path = require('path');

const filePath = 'src/app/instructor/[sessionId]/analysis/page.tsx';
const content = fs.readFileSync(filePath, 'utf8');

console.log('File length:', content.length);
console.log('Braces {:', (content.match(/\{/g) || []).length);
console.log('Braces }:', (content.match(/\}/g) || []).length);
console.log('Parens (:', (content.match(/\(/g) || []).length);
console.log('Parens ):', (content.match(/\)/g) || []).length);
console.log('Frag <>:', (content.match(/<>/g) || []).length);
console.log('Frag </>:', (content.match(/<\/>/g) || []).length);
console.log('Div <div>:', (content.match(/<div/g) || []).length);
console.log('Div </div>:', (content.match(/<\/div>/g) || []).length);
